includes/discovery/bgp-peers/dell-os10.inc.php

```diff
@@ -40,7 +42,7 @@
         unset($bgpPeersCache);
 
         foreach ($bgpPeers as $vrfInstance => $peer) {
-            $vrfId = dbFetchCell('SELECT vrf_id from `vrfs` WHERE vrf_oid = ?', [$vrfInstance]);
+            $vrfId = Vrf::select('vrf_id')->where('device_id', '=', $device['device_id'])->where('vrf_oid', '=', $vrfInstance)->first();
```

Comment:
There is a shortcut for this `->where('device_id', '=', $device['device_id'])`

`DeviceCache::getPrimary()->vrfs()->firstWhere('vrf_oid', $vrfInstance);`

(also = is the default operation, so you can omit it.

Reply:
Thx for tip. Will apply it.